### DISCUSSION TITLE
Issuer certificate issue for Prisma generate or prisma -v command

### DISCUSSION THREAD

**Discussion by aryan0-hash** [prisma/prisma#21599] [Q&A]
### Question

Whenever I am trying to run "prisma generate or prisma -v" I am getting error : Request to https://binaries.prisma.sh/all_commits/0362da...... failed,reason : unable to get local issuer certificate

I already tried setting strictSSL to false and NODE_TLS_REJECT_UNAUTHORIZED=0 and NODE_EXTRA_CA_CERTS = cert path but not working. I'm using next.js and typescript with prisma. All version are upto dated with prisma.

I am working in a corporate network. Is there any solution in that respect? Don't recommend github discussion pages i hv tried all none of are working. Please one knows please answer me. I got struck for a week please answer as soon as possible.

Operating system:Windows 10
My company is using zscaler for security purposes.
Database:- PostgreSQL

### Expected behavior (optional)

It should download prisma engines and can able to get version of prisma. 

Environment variables loaded from .env
prisma               : 2.21.0-dev.4
@prisma/client       : 2.21.0-dev.4
Current platform     : windows
Query Engine         : query-engine 2fb8f444d9cdf7c0beee7b041194b42d7a9ce1e6 (at C:\Users\veroh\AppData\Roaming\npm\node_modules\@prisma\cli\query-engine-windows.exe)
Migration Engine     : migration-engine-cli 2fb8f444d9cdf7c0beee7b041194b42d7a9ce1e6 (at C:\Users\veroh\AppData\Roaming\npm\node_modules\@prisma\cli\migration-engine-windows.exe)
Format Binary        : prisma-fmt 60ba6551f29b17d7d6ce479e5733c70d9c00860e (at node_modules\@prisma\engines\prisma-fmt-windows.exe)
Default Engines Hash : 60ba6551f29b17d7d6ce479e5733c70d9c00860e
Studio               : 0.365.0

### Information about Prisma Schema, Client Queries and Environment (optional)

```prisma
// Add your schema.prisma
```

```ts
// Add any relevant Prisma Client queries here
```

- OS: 
- Database: 
- Node.js version: 

- Run `prisma -v` to see your Prisma version and paste it

**Comment by ludralph** [comment]
Hi @aryan0-hash 👋 

Thank you for raising this question. 

Can you please confirm if running the command `prisma generate` or `prisma -v` works outside the company's network?

If it works then it's most likely an issue with the firewall setup of Zscaler and you may need to contact your system administrator.

**Reply by janpio** [reply]
Additionally, you should _really_ upgrade from this ancient version `2.21.0-dev.4`. Very possible that we have implemented a fix to this problem in the last 2.5 years.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
